neo/Ledger/Blockchain.cs

```diff
@@ -228,7 +228,7 @@ private void OnImport(IEnumerable<Block> blocks)
             foreach (Block block in blocks)
             {
                 if (block.Index <= Height) continue;
-                if (block.Index != Height + 1)
+                if (block.Index != Height + 1 || block.PrevHash != CurrentBlockHash)
```

Comment:
We can check the first hash only, instead of check it for every block.

Reply:
Exactly, I was going to propose this as well, @erikzhang, connected minds.

Reply:
What if  there is wrong block in middle of chain data package? Should we move this check logic in ImporBlocksPlugin?

Reply:
I think that we should verify every blocks, we need to ensure that the signature is the expected one, slow, but secure.